modules/base/csv.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2018 The Gitea Authors. All rights reserved.
```

Comment:
Why no still keep these code in package `modules/markup/csv/`?

Reply:
I thought the csv reader creation logic shouldn't be coupled with the render part.

Reply:
I'm OK with another package but put them in `base` package seems not suitable. Maybe we can create a new package named `modules/csv` or similar to do that.

Reply:
I moved the code to a new csv package.